package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "@bazel/bazelisk",
+    "version": "0.0.0-PLACEHOLDER",
```

Comment:
How is the "version" here replaced with the correct version?

Reply:
It's a feature of the pkg_npm rule: https://github.com/bazelbuild/rules_nodejs/blob/master/docs/Built-ins.md#replace_with_version